registry/default/blocks/post-page/components/nsfw-warning.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { EyeOff } from "lucide-react";
+
+import { cn } from "@/lib/utils";
+import { Button } from "@/registry/default/ui/button";
+import { Card } from "@/registry/default/ui/card";
+import { Stack } from "@/registry/default/ui/stack";
+import { Typography } from "@/registry/default/ui/typography";
+
+const NsfwWarning = ({
```

Comment:
On the call myself, you and Basma had we said we aren't tackling NSFW now. But you end up implementing it and this is a pattern I'm observing about how you work. Why ?